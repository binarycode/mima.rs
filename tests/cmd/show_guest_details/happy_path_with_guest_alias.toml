args = 'guest zero'
status.code = 0
