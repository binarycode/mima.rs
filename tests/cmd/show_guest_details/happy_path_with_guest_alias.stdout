GUEST  ID    BOOTED  SPICE  MEMORY  CORES  DESCRIPTION
       zero  true    5901   8192    4      Test Virtual Machine

DISKS  LABEL  SIZE  PATH
       sda    20    /mnt/mima/zero/sda.qcow2
       sdb    100   /mnt/mima/zero/sdb.qcow2

NETWORK INTERFACES  NETWORK  MODEL                            MAC                TAP
                    pub      virtio-net-pci-non-transitional  52:54:00:00:00:10  mima-pub-zero
                    mgt      e1000e                           52:54:00:00:09:10  mima-mgt-zero
